assets/css/admin-custom.css

```diff
@@ -111,7 +111,7 @@ div.question_boolean_fields { margin-bottom: 10px; }
 
 .post-type-course .wp-list-table .button-link {
 	color: #2C3338;
-	display: inline-block;
+	display: block;
```

Comment:
Changing the CSS here breaks _Quick Edit_:

![Screen Shot 2022-01-26 at 11 08 44 AM](https://user-images.githubusercontent.com/1190420/151200740-fc2fe461-1e64-46f2-a8a4-93a2c50600a4.jpg)

Reply:
Fixed in 70a7a653302e331d52142833899f5fecfeee428f and 09cb1f7f877f97430a0eafb7714124f75c49bd7e. Decided to use a custom class. Let me know if you have a better name. 🙂